src/lib/react/React.hx

```diff
@@ -28,6 +28,12 @@ extern class React
 	**/
 	public static function isValidElement(object:Dynamic):Bool;
 
+	/**
+		https://reactjs.org/docs/context.html#reactcreatecontext
+		Note: this API has been introduced in React 16.3
+	**/
+	public static function createContext<T>(?defaultValue:T):ReactContext<T>;
```

Comment:
`createContext` also accepts an optional second argument, `calculateChangedBits` of type `T->T->Int`.

https://github.com/facebook/react/blob/master/packages/react/src/ReactContext.js#L19
(I know, it's not documented)